scripts/run.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
```

Comment:
This file requires Python >= 3.6 (at least for f strings). The shebang should reflect this and refer to at leas python3.

Coding style: Generally, we should not split an instruction across several lines unless we reach the line length limit.